chia/cmds/wallet_funcs.py

```diff
@@ -68,6 +107,45 @@ async def get_transactions(args: dict, wallet_client: WalletRpcClient, fingerpri
                 break
 
 
+async def get_csv(args: dict, wallet_client: WalletRpcClient, fingerprint: int) -> None:
+    wallet_id = args["id"]
+    offset = args["offset"]
+    file_path = args["file"]
+    date_fmt = args["date_fmt"]
+    delim = args["delim"]
+
+    txs: List[TransactionRecord] = await wallet_client.get_csv(wallet_id)
+    config = load_config(DEFAULT_ROOT_PATH, "config.yaml", SERVICE_NAME)
+    name = config["network_overrides"]["config"][config["selected_network"]]["address_prefix"]
+    if len(txs) == 0:
+        print("There are no transactions to this address")
+        return
+
+    header_list = [
+        "wallet_id",
+        "date_time",
+        "tx_type",
+        "status",
+        "height_confirmed",
+        "tx_name",
+        "amount",
+        "fee",
+        "to_address",
+    ]
+    file_header = delim.join(map(str, header_list))
```

Comment:
the `str` cast is redundant here

Reply:
removing str also fails mypy lint

Reply:
There should be no issue with
```python
delim.join(header_list)
```
in fact you can test it locally.
I imagine the test fail is because of something else.

Sidenote: I'd all caps the list if it's just a constant, `HEADER_LIST`.

Reply:
Lol... you are correct I was doing:

```delim.join(map(header_list))```

I will adjust this per your suggestion